src/screens/DashboardScreen.tsx

```diff
@@ -62,7 +62,7 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
   return (
     <DashboardScreenStyled
       refreshControl={
-        <RefreshSpinner refreshing={isLoading} onRefresh={refreshData} progressViewOffset={headerHeight} />
+        <RefreshSpinner refreshing={isLoading} onRefresh={refreshData} progressViewOffset={headerHeight || 170} />
```

Comment:
Quick and dirty, but I had to add this because useHeaderHeight doesn't seem to work in the DashboardScreen anymore.